_Week5/group1/one_way_anova.do

```diff
@@ -2,9 +2,6 @@
 Week 5 Assignment
 Program .do file */
 
-* Explanation of ANOVA: https://sphweb.bumc.bu.edu/otlt/MPH-Modules/BS/BS704_HypothesisTesting-ANOVA/BS704_HypothesisTesting-Anova3.html
```

Comment:
You may want to consider moving this to the readme file because I'd assume someone would go through the documentation before running the program. If they're not familiar with ANOVA, they might need to look up the explanation themselves in that case.

Reply:
Good catch! We included it in an earlier version but removed these two commented lines in the latest version. Definitely agree it doesn't belong in the .do file!